src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -83,14 +104,32 @@ public JedisCluster(HostAndPort node, final JedisClientConfig clientConfig, int
     this(Collections.singleton(node), clientConfig, maxAttempts, poolConfig);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances.
+   * Here, the default Timeout of {@value #DEFAULT_TIMEOUT} ms is being used with {@value  #DEFAULT_MAX_ATTEMPTS} maximum attempts.
+   * @param nodes Hosts to connect to.
+   */
   public JedisCluster(Set<HostAndPort> nodes) {
     this(nodes, DEFAULT_TIMEOUT);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances.
+   * Here, the default of {@value  #DEFAULT_MAX_ATTEMPTS} maximum attempts is being used.
+   * @param nodes Hosts to connect to.
+   * @param timeout Timeout in milliseconds.
+   */
   public JedisCluster(Set<HostAndPort> nodes, int timeout) {
     this(nodes, DefaultJedisClientConfig.builder().timeoutMillis(timeout).build());
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances. <br>
```

Comment:
Here, one of the multiples nodes/instances is used as an entry point (or to make the first contact) with the cluster.